### PR TITLE
Mmt 2823 - Fix tests failures.

### PR DESCRIPTION
**Pull request by bademiluyi**
Fix tests failures.

### CODE REVIEW

**Comment by bademiluyi** [comment]
Updated. Ready for review.

File: spec/features/collection_drafts/publish_collection_draft_spec.rb [hunk 1]
```diff
@@ -102,15 +101,15 @@
             click_on 'Done'
           end
           click_on 'Publish'
-          find('label.tab-label', text: 'Citation Information').click
+          #find('label.tab-label', text: 'Citation Information').click
```
Comment (cgokey) [inline spec/features/collection_drafts/publish_collection_draft_spec.rb:104]: remove this

**Review by cgokey — APPROVED** [review]

**Review by tle1989 — APPROVED** [review]

**Review by cgokey — CHANGES_REQUESTED** [review]
Still seeing the =>

File: spec/features/collection_drafts/auto_populate_metadata_dates_spec.rb [hunk 2]
```diff
@@ -14,13 +14,11 @@
...
       click_on 'Publish'
       wait_for_cmr
-
-      find('.tab-label', text: 'Additional Information').click
     end
 
     it 'displays the create date on the collection page' do
-      within '.metadata-dates-table' do
-        expect(page).to have_content("Creation #{today_string}")
+      within '#metadata-preview' do
```
Comment (cgokey) [inline spec/features/collection_drafts/auto_populate_metadata_dates_spec.rb:20]: This still needs to be fixed.

File: spec/features/collection_drafts/auto_populate_metadata_dates_spec.rb [hunk 3]
```diff
@@ -14,13 +14,11 @@
...
+        expect(page).to have_content('"MetadataDates"=>[{"Type"=>"CREATE", "Date"=>"')
```
Comment (cgokey) [inline spec/features/collection_drafts/auto_populate_metadata_dates_spec.rb:21]: Change the =>

**Review by cgokey — APPROVED** [review]

**Pull request merged**